index.html

```diff
@@ -70,9 +70,207 @@ <h1>
+              <h4 class="title">Multi-Post Stories Gain+Glory</h4>
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>Html</li>
+              </ul>
+              <button>See Project</button>
+            </section>
+          </article>
+          <article class="card">
+            <section class="content">
+              <h4 class="title">Multi-Post Stories Gain+Glory</h4>
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>Html</li>
+              </ul>
+              <button>See Project</button>
+            </section>
+          </article>
+          <article class="card">
+            <section class="content">
+              <h4 class="title">Multi-Post Stories Gain+Glory</h4>
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>Html</li>
+              </ul>
+              <button>See Project</button>
+            </section>
+          </article>
+          <article class="card">
+            <section class="content">
+              <h4 class="title">Multi-Post Stories Gain+Glory</h4>
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>Html</li>
+              </ul>
+              <button>See Project</button>
+            </section>
+          </article>
+          <article class="card">
+            <section class="content">
+              <h4 class="title">Multi-Post Stories Gain+Glory</h4>
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>Html</li>
+              </ul>
+              <button>See Project</button>
+            </section>
+          </article>
+        </div>
+      </section>
+      <!-- about section -->
+      <section class="about-section">
+        <article class="bio-container">
+          <h1 class="heading">About me</h1>
+          <p class="desc">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+          <button class="btn">Get My Resume</button>
+        </article>
+
+        <section class="bio-details">
+          <article class="card">
+            <div class="icon-container">
+              <img src="./assets/language.png" alt="language-icon" />
+            </div>
+            <section class="content">
+              <h4 class="title">Languages</h4>
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>Html</li>
```

Comment:
<img width="1003" alt="2" src="https://user-images.githubusercontent.com/77843050/220865503-37e97174-a35c-4a04-84d5-11aa0a403eaa.png">

- [ ] Consider fixing the padding and margin as I highlighted here, it is required to have the same as in the original design.

Reply:
@joskalenda Hello bro, kindly check it out for me, I am done with the necessary changes you asked me to make.